test/repl.c

```diff
@@ -0,0 +1,161 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include "at_client.h"
+
+int main()
+{
+
+    int ret = 1;
+
+    // initialize buffer to use throughout program
+
+    size_t recvlen = 32768;
+    unsigned char *recv = malloc(sizeof(unsigned char) * recvlen);
+    size_t *olen = malloc(sizeof(size_t));
+
+    // initialize atkeys
+
+    const char *keyspath = "/Users/jeremytubongbanua/.atsign/keys/@jeremy_0_key.atKeys";
```

Comment:
Should move these hardcoded values to an environment / config file later (which is gitignored)